seed/challenges/02-javascript-algorithms-and-data-structures/debugging.json

```diff
@@ -68,6 +68,42 @@
       "translations": {}
     },
     {
+      "id": "587d7b83367417b2b2512b37",
+      "title": "Clarifying the In-Browser Console Log",
+      "description": [
+        "In an ordinary Javascript file loaded and run in your browser a <code>console.log()</code> will print exactly what you tell it to print to the console. In your in-browser text editor the process is slightly different and can be confusing at first. Any value passed to a <code>console.log</code> runs each set of test data as well as your function call.",
+        "This lends itself to some interesting behavior and might trip you up in the begining, because a line that you expect to see only once will run all of the variables presented in the tests as well as the in-browser text editor printing to the console.",
+        "If you would like to see only your single output and not have to worry about running through the test cycles you can use <code>console.clear()</code>",
+        "For more console based functions check out:",
+        "&nbsp<a href=\"https://developer.mozilla.org/en-US/docs/Web/API/Console\">MDN Console Reference</a>"
```

Comment:
Add an 'instructions' area by including a paragraph with a horizontal rule (`<hr>`), then a new paragraph that says "Use `console.log() to print the variables in the code where indicated."